tgui/packages/tgui/styles/components/Tabs.scss

```diff
@@ -10,21 +10,44 @@
 $color-default: color.scale(colors.fg(colors.$primary), $lightness: 75%) !default;
 $text-color: rgba(255, 255, 255, 0.5) !default;
 $text-color-selected: color.scale($color-default, $lightness: 25%) !default;
+$tab-color: transparent !default;
+$tab-color-hovered: rgba(255, 255, 255, 0.075) !default;
+$tab-color-selected: rgba(255, 255, 255, 0.125) !default;
 $border-radius: base.$border-radius !default;
 $fg-map: colors.$fg-map !default;
 
 .Tabs {
   display: flex;
   align-items: stretch;
   overflow: hidden;
+  background-color: base.$color-bg-section;
```

Comment:
Are there really no cases where we would want a tab set to appear outside a level 1 section? It might be nice to have the option

Reply:
😔

Reply:
> Right now, it looks better that way

You're probably right, I guess we can see how this works out